optuna_dashboard/EP.py

```diff
@@ -0,0 +1,302 @@
+#%%
+
+import numpy as np
```

Comment:
Could you remove this file?

Reply:
Sorry, I added it unintentionally. I removed it.